packages/angular-parser/src/index.ts

```diff
@@ -244,29 +243,15 @@ const visitor = {
       .replace(/^\[attr\./, '')
       // remove leading `*`, `[]` and `()` wrapper
       .replace(/[()*[\]]/g, '')
-    // remove leading `#`
-    const plainName = potentialName.replace(/#/g, '')
-    const potentialValue = _value.replace(/(^{\s*{)|(}\s*}$)/g, '')
 
     node.potentialName = potentialName
-    node.isInvalid =
```

Comment:
Why `isInvalid` property removed?

Reply:
@JounQin 

Sorry, I forgot to improve the process on this condition.

I changed the concept of the invalid attribute syntax to throwing an error. The error is caught by the core linter engine then it reports a violation as the parse error. 

https://github.com/markuplint/markuplint/commit/5a0d45e33a460ca6b98349e0e7b6737437150283